src/main/java/com/google/step/servlets/JsonFormatter.java

```diff
@@ -1,17 +1,48 @@
 package com.google.step.servlets;
 
 import com.google.gson.Gson;
+import com.google.step.model.Comment;
 import com.google.step.model.Deal;
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 public class JsonFormatter {
+  public static String getCommentsJson(List<Comment> comments) {
+    if (comments.size() == 0) {
```

Comment:
I think the code below covers this case, so there is no need to explicitly cover it